apps/kirjs/src/app/modules/svg-race/player/player.component.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnInit,
+  Output,
+  SimpleChanges,
+  ViewChild
+} from '@angular/core';
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: '[kirjs-player]',
+  templateUrl: './player.component.html',
+  styleUrls: ['./player.component.css']
+})
+export class PlayerComponent implements OnInit, AfterViewInit, OnChanges {
+  @ViewChild('guess', {static: true}) guess: ElementRef;
+  @Input() path;
+  @Input() d = '';
```

Comment:
use a better name?

Reply:
Wanna keep it consistent with the attribute name. Blame svg lol